README.md

```diff
@@ -86,12 +86,13 @@ Once complete, your final assembly should look like this:
 
 The Arduino firmware (adanet-co2-monitor.ino) has been tested using the following software (and corresponding versions):
 
-- Arduino IDE (v1.8.19)
+- Arduino IDE (v1.8.19 or v2.0.0)
 - Adafruit DPS310 (v1.1.1)
 - Adafruit EPD (v4.5.0)
 - Adafruit LC709203F (v1.3.1)
 - ESP32 Arduino (v2.0.5)
 - Sensirion I2C SCD4x (v0.3.1)
+- Adafruit NeoPixel (v1.10.6)
```

Comment:
Ah, I think the issue here is just `#include <Adafruit_NeoPixel.h>` in both .ino files (but not used). Just remove those lines instead?

Reply:
Yep, that was it, fixed.